src/api/register.js

```diff
@@ -37,6 +40,29 @@ router.post("/register", async function(req, res) {
   }
 
   const jwt_token = signAuthJWT(user._id, user.password);
+
+  if (usingGmail) {
+    // using gmail so should send verification email
+    generateAndCommitPIN();
+    const body = {
+      from: "hack4impact.infra@gmail.com",
+      to: user.email,
+      subject: "Forgot Password",
+      text: "Enter the following pin on the reset page: " + user.pin
+    };
+    try {
+      await sendMail(body);
+    } catch (e) {
+      // TODO: discuss whether or not we want to register the user even if the email doesn't send
```

Comment:
Since they can't verify the email, I think we shouldn't bother even making the user in the first place. It prevents the db from being filled up with a bunch of users that can never be verified.

Reply:
That's true. Unless we have another `resendVerification` endpoint.

Reply:
That's a good point. The only way I can see around that would be having a `POST /resendVerification` endpoint.

Reply:
Thats actually a really good point, let's actually create an endpoint for this!

Reply:
Awesome - will open up a separate PR (unless you'd like me to do it as part of this one).